models/model.js

```diff
@@ -52,3 +52,18 @@ exports.selectAllArticles = async () => {
 
   return articles;
 };
+
+exports.selectCommentsById = async (id) => {
+  const { rows: comments } = await db.query(
+    "SELECT * FROM comments WHERE article_id = $1",
+    [id]
+  );
+  if (comments.length === 0) {
+    return Promise.reject({
+      status: 404,
+      msg: `No user found for article_id: ${id}`,
+    });
```

Comment:
I'm not sure what this error is finding. If the article id exists but there are no comments associated in the databaase, has an error occured?

Reply:
It also doesn't look like this was tested.

Reply:
I see what this error is for now. I would argue that you're trying to combine two possible outcomes into one test - one of which is an error and the other which isn't. What happens if the id does exist but has no comments vs what is the id doesn't exist?